Lib/test/test_traceback.py

```diff
@@ -2979,8 +2979,8 @@ class MyClass:
             with self.subTest(name=name):
                 actual = self.get_suggestion(MyClass, name)
                 self.assertNotIn("you mean", actual)
```

Comment:
Maybe check for ``Did you mean``? Same remark for the other test.

Other test_traceback tests check for this (a little bit longer) pattern.